src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -35,7 +35,7 @@ public void isSamePerson() {
         // same name, all other attributes different -> returns true
```

Comment:
May consider changing this comment to return false, so that it is consistent with the code